test/library/library.go

```diff
@@ -0,0 +1,35 @@
+package library
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+	"path"
+	"regexp"
+	"runtime"
+	"testing"
+	"time"
+)
+
+var (
+	waitPollInterval = time.Second
+	waitPollTimeout  = 10 * time.Minute
+)
+
+func GenerateNameForTest(t *testing.T, prefix string) string {
```

Comment:
which kind of name?

Reply:
Added godoc.

Reply:
randomResourceName would be a good func name. says everything one needs to know.